src/base/bittorrent/session.cpp

```diff
@@ -1093,6 +1176,61 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
     settingsPack.set_bool(libt::settings_pack::enable_lsd, isLSDEnabled());
 }
 
+void Session::configurePeerClasses()
+{
+    libt::ip_filter f;
+    f.add_rule(libt::address_v4::from_string("0.0.0.0")
+               , libt::address_v4::from_string("255.255.255.255")
+               , 1 << libt::session::global_peer_class_id);
+#if TORRENT_USE_IPV6
+    f.add_rule(libt::address_v6::from_string("::0")
+               , libt::address_v6::from_string("ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff")
+               , 1 << libt::session::global_peer_class_id);
+#endif
+    if (ignoreLimitsOnLAN()) {
+        f.add_rule(libt::address_v4::from_string("10.0.0.0")
+                   , libt::address_v4::from_string("10.255.255.255")
+                   , 1 << libt::session::local_peer_class_id);
+        f.add_rule(libt::address_v4::from_string("172.16.0.0")
+                   , libt::address_v4::from_string("172.16.255.255")
```

Comment:
is this range from the old code?
it's wrong according to: https://en.wikipedia.org/wiki/Private_network